### PR TITLE
[CUTLASS] Fix hardcoded include path and logic for profile_all = False case

### PR DESCRIPTION
**Pull request by masahi**
* The path to the cutlass dir is hardcoded with a relative path, the test only works when run from `test/python/contrib` :disappointed: 
* When `profile_all = False`, only one kernel is supposed to be compiled, but this logic was busted when I added `compile_all` function.

@comaniac @Laurawly

### CODE REVIEW

File: python/tvm/contrib/cutlass/build.py [hunk 1]
```diff
@@ -16,11 +16,17 @@
...
 """Driver for partitioning and building a Relay module for CUTLASS offload."""
+import os
 import tvm
 from tvm import runtime, relay
 from .gen_gemm import CutlassGemmProfiler
 
 
+def _get_cutlass_path():
+    tvm_root = os.path.join(os.path.dirname(os.path.realpath(__file__)), "../../../../")
+    return os.path.join(tvm_root, "3rdparty/cutlass")
```
[the remark this answers is not in the document]
Comment (masahi) [reply python/tvm/contrib/cutlass/build.py:27]: hmm right... I don't have a good solution for that problem.

Can we say we don't support cutlass on such environments?

[the remark this answers is not in the document]
Comment (comaniac) [reply python/tvm/contrib/cutlass/build.py]: Yeah I think it's fine to treat it as a limitation for now as long as we have a proper message when error out.

[the remark this answers is not in the document]
Comment (masahi) [reply python/tvm/contrib/cutlass/build.py]: ok added a msg saying source build is required for cutlass.

**Review by comaniac — APPROVED** [review]
LGTM. Thanks

**Comment by masahi** [comment]
Thanks @comaniac

**Pull request merged**